src/styles/blocks/card.scss

```diff
@@ -0,0 +1,77 @@
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: $card-width;
+  border-radius: 5px;
+  background-color: $color-white;
+  border: 1px solid $border-color;
+  color: $color-dark;
+  padding: 32px 16px 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-weight: 500;
+    @extend %large-text;
+    margin-bottom: 4px;
+  }
+
+  &__description {
+    @extend %small-text;
+    margin-bottom: 16px;
+    color: $color-additional;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    padding-bottom: 16px;
+    line-height: 18px;
+
+    &-name {
+      color: $color-additional;
+      @extend %large-text;
+    }
+
+    &-value {
+      font-weight: bold;
+      font-size: 16px;
+      color: $color-dark;
+    }
+  }
+
+  &__link {
+    padding: 11px 70px;
+    font-weight: bold;
```

Comment:
Actually it's better to use numerical values(in this case 700) and also you can add variable for this value

Reply:
it mieans the same